samples/kinesis_file_gstreamer_sample_app.cpp

```diff
@@ -0,0 +1,797 @@
+#include <gst/gst.h>
+#include <gst/app/gstappsink.h>
+#include <string.h>
+#include <chrono>
+#include <Logger.h>
+#include "KinesisVideoProducer.h"
+#include <vector>
+#include <stdlib.h>
+#include <mutex>
+#include <IotCertCredentialProvider.h>
+
+using namespace std;
+using namespace com::amazonaws::kinesis::video;
+using namespace log4cplus;
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+int gstreamer_init(int, char **);
+
+#ifdef __cplusplus
+}
+#endif
+
+LOGGER_TAG("com.amazonaws.kinesis.video.gstreamer");
```

Comment:
Consider changing the tag to something shorter and specific to the app. Can simply be the app name

Reply:
Makes sense!